src/main/java/co/rsk/federate/config/FedNodeSystemProperties.java

```diff
@@ -20,6 +20,10 @@ public boolean isFederatorEnabled() {
         return configFromFiles.hasPath("federator.enabled") && configFromFiles.getBoolean("federator.enabled");
     }
 
+    public boolean isPegoutEnabled() {
+        return !configFromFiles.hasPath("federator.pegout.enabled") || configFromFiles.getBoolean("federator.pegout.enabled");
```

Comment:
```suggestion
        return configFromFiles.hasPath("federator.pegout.enabled") && configFromFiles.getBoolean("federator.pegout.enabled");
```
Isnt This Simpler To Read?

Reply:
Nice K! I like yours. Looks simpler to read for me too.